packages/core/src/runtime/Hook.ts

```diff
@@ -0,0 +1,31 @@
+export declare function afterAll(fnc: HookFn, timeout?: number): void
+export declare function afterEach(fnc: HookFn, timeout?: number): void
+export declare function beforeAll(fnc: HookFn, timeout?: number): void
+export declare function beforeEach(fnc: HookFn, timeout?: number): void
+
+export type HookFn = (this: void) => any
```

Comment:
I think we should allow passing `browser` variable in case if the user wants to do something/checking element
```suggestion
export type HookFn = (this: void, browser: Browser) => Promise<any>
```